File: backend/src/main/java/ipeps/pwd/wallet/entity/payload/TimesheetCreatePayload.java
package ipeps.pwd.wallet.entity.payload;

import ipeps.pwd.wallet.entity.Contract;
import ipeps.pwd.wallet.entity.Employee;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
@Data
@NoArgsConstructor
@AllArgsConstructor
public class TimesheetCreatePayload {
    Date Start_date;
    Date Start_hours;
    Date End_hours;
    String Comment;
    String Timesheet_type;
    Employee Employee;
    Contract Contract;


}

File: WalletApp/src/app/modules/timesheet/model/payload/timesheet.create.payload.ts
import {Employee} from "../../../employee/model/business/employee";
import {Contract} from "../../../contract/model/business/contract";
import {EmployeeDto} from "../../../employee/model/dto/employee.dto";
import {ContractDto} from "../../../contract/model/dto/contract.dto";

export interface TimesheetCreatePayload {
  employee:EmployeeDto;
  contract:ContractDto;
  start_date:string;
  start_hours:string;
  end_hours:string;
  comment:string;
  timesheet_type:string;

}
